packages/client/src/pages/forum/mocData.ts

```diff
@@ -3,50 +3,165 @@ import { ForumData } from 'pages/forum/types';
 export const mocForumData: ForumData = [
```

Comment:
moc - такого слова нет, есть только mock

```suggestion
export const mockForumData: ForumData = [
```

Reply:
Поправил.